site/en/docs/extensions/reference/offscreen/index.md

```diff
@@ -2,4 +2,34 @@
 api: offscreen
 ---
 
-<!-- Intentionally blank -->
+## Manifest
+You must declare the "offscreen" permission in the [extension manifest][1] to use the Offscreen API. For example:
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "offscreen"
+  ],
+  ...
+}
+```
+Pages loaded as offscreen documents are handled differently from other types of extensions pages. The extension's permissions carry over to offscreen documents, but extension API access is heavily limited. Currently, an offscreen document can only use the `chrome.runtime` APIs to send and receive messages; all other extension APIs are not exposed. Other notable differences between offscreen documents and normal pages are as follows:
+
+* Offscreen document's URL must be a static HTML file bundled with the extension.
+* Offscreen documents cannot be focused.
+* Offscreen documents cannot have their `opener` property set using the [`chrome.windows` API](docs/extensions/reference/windows/) method `windows.setSelfAsOpener`.
+
+## Examples
+The following methods create and close an offscreen document. Only a single Document can be open at a time. 
+
+```js
+chrome.offscreen.createDocument({
+  url: chrome.runtime.getURL('off_screen.html'),
+  reasons: ['CLIPBOARD'],
```

Comment:
It'd be nice to:

- Provide a list of each of the possible reasons, since developers probably won't know where to find those in the code (see [here](https://source.chromium.org/chromium/chromium/src/+/main:extensions/browser/api/offscreen/lifetime_enforcer_factories.cc;drc=503535015a7b373cc6185c69c991e01fda5da571;l=80?q=kReasonAndFactoryMethodPairs&ss=chromium)).
- Document that the browser may take steps to enforce the use (e.g requiring audio to play if audio playback is the justification).

I think a small tweak for each of these should be a blocker for merging because otherwise developers might simply copy this snippet and not update the reasons. We can probably just have a single line for point 2 for now though and have a follow-up to explicitly document how Chrome enforces each.